projects/kubernetes/image-cache/mkversions

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+repo=gcr.io/google_containers
+kube_version=v1.7.6
+kube_dns_version=1.14.4
+pause_version=3.0
+etcd_version=3.0.17
+
+common="
+	kube-proxy-amd64:$kube_version
+	k8s-dns-sidecar-amd64:$kube_dns_version
+	k8s-dns-kube-dns-amd64:$kube_dns_version
+	k8s-dns-dnsmasq-nanny-amd64:$kube_dns_version
+	pause-amd64:$pause_version"
+
+control="
+	kube-apiserver-amd64:$kube_version
+	kube-controller-manager-amd64:$kube_version
+	kube-scheduler-amd64:$kube_version
+	etcd-amd64:$etcd_version"
+
+for i in $common $control ; do
+	docker image pull "$repo/$i" 1>&2
+done
+
+oi() {
+	local i="$1"
+	digest=$(docker image inspect "$repo/$i" | jq -r '.[].RepoDigests[0]')
```

Comment:
`docker inspect --format '{{index .RepoDigests 0}}' "$repo/$i"` might be simpler

Reply:
It would indeed, thanks.

Since this code is added before the "Bump yml" commit and it's a pain to rebuild I'm going to fix this at the head of the branch rather than fold it into the right place.